test/dynamo/test_dynamo.py

```diff
@@ -671,6 +676,22 @@ def foo(x):
     self.assertEqual(expected.dtype, actual.dtype)
     self.assertEqual(expected.device, actual.device)
 
+  def test_return_expand(self):
+    _set_view_replay_for_aliased_outputs()
```

Comment:
do we want to leave this options to always true througout the test? If the intention is to always set this to True for PyTorch/XLA, you should set it in https://github.com/pytorch/xla/blob/master/torch_xla/__init__.py .